scripts/download-stats.ts

```diff
@@ -0,0 +1,152 @@
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import { Parser } from 'json2csv';

```

Comment:
See if papaparse (already a dependency) works for these needs and remove json2csv.